metacat-common-server/src/main/java/com/netflix/metacat/common/server/converter/ConverterUtil.java

```diff
@@ -81,6 +83,7 @@ protected void configure() {
                 mapping(PartitionDto.class, PartitionInfo.class)
                     .fields("name", "name", FieldsMappingOptions.copyByReference());
                 mapping(AuditDto.class, AuditInfo.class);
+                mapping(HiveVirtualViewDto.class, HiveVirtualViewInfo.class);
                 mapping(StorageDto.class, StorageInfo.class);
```

Comment:
Somehow this mapping isn't working in the TableDto <-> TableInfo. Added a get around.